docs/usage.md

```diff
@@ -128,6 +128,38 @@ to the root directory of the EUKulele databases.
 
 > Please, check the [EUKulele documentation](https://eukulele.readthedocs.io/en/latest/#) for more information about the databases.
 
+> :warning: There is a bug in EUKulele program that doesn't allow you to download some databases properly, check [EUKulele issue](https://github.com/AlexanderLabWHOI/EUKulele/issues/60). Meanwhile the developers are fixing this bug, we reccomand to download the database manually. To do so, follow these steps:
```

Comment:
```suggestion
> :warning: There is currently a bug in the EUKulele program so that some databases properly do not download properly, check [EUKulele issue](https://github.com/AlexanderLabWHOI/EUKulele/issues/60). Until the developers have fixed this bug, we reccomend to download the database manually. To do so, follow these steps:
```

Reply:
Do you know which databases do not work? If so, it would be great to enumerate them here. Perhaps in parentheses after "properly": `properly (MMETSP, GTDB, NN)`.

Reply:
I think that only phyloDB is actually working